syslog-ng/main.c

```diff
@@ -217,6 +228,7 @@ main(int argc, char *argv[])
   g_process_set_argv_space(argc, (gchar **) argv);
 
   resolved_configurable_paths_init(&resolvedConfigurablePaths);
+  __resolve_paths_in_help_texts();
```

Comment:
Can't we just move `syslogng_options` to the stack (into a function call that is responsible for the initialization of the command line options), and call `get_installation_path_for()` directly on the `PATH_SYSLOG_NG_CONF`, `SYSLOG_NG_MODULE_PATH`, etc. constants inside the option description?

If you do not prefer that method, can we just remove the double dash prefix of `__resolve_paths_in_help_texts` -> `_resolve_paths_in_help_texts`?